openedx/features/student_account/helpers.py

```diff
@@ -163,3 +165,13 @@ def check_and_add_third_party_params(request, params):
             params['provider'] = running_pipeline.get('backend')
 
         params['access_token'] = running_pipeline['kwargs']['response']['access_token']
+
+
+def get_registration_countries():
+    countries_dict = {'all_countries': sorted(COUNTRIES.items(), key=operator.itemgetter(1)),
+                      'top_countries': []}
+    if TOP_REGISTRATION_COUNTRIES:
+        countries_dict.update({'top_countries': TOP_REGISTRATION_COUNTRIES})
+        return countries_dict
+    else:
```

Comment:
Let's please not use `else` and simply return countries.

Reply:
Approach changed, this is not valid anymore.